src/languages/postgresql/postgresql.formatter.ts

```diff
@@ -18,6 +18,7 @@ const reservedClauses = expandPhrases([
   'LIMIT',
   'OFFSET',
   'FETCH {FIRST | NEXT}',
+  'FOR {UPDATE | NO KEY UPDATE | SHARE | KEY SHARE} [OF]',
```

Comment:
This was taken straight from the Postgres docs: https://www.postgresql.org/docs/current/sql-select.html